app/packages/looker/src/state.ts

```diff
@@ -23,6 +23,7 @@ export interface Coloring {
 }
 
 export type OrthogrpahicProjectionMetadata = {
+  _id: string;
```

Comment:
[OrthographicProjectionMetadata](https://github.com/voxel51/fiftyone/blob/130cf6022260809a02c269561cd2f154709b7d54/fiftyone/utils/utils3d.py#L413-L430) doesn't have an `id` attribute, so we should not add this

Reply:
we have to have it because it gives the specific id which is expected by downstream constants and is also what enables filtering and such to work

Reply:
Is this done for the regular `Metadata` classes too? They don't have `id` attributes either

Reply:
It's not called Metadata, but this is it's equivalent for non-ortho proj samples: https://github.com/voxel51/fiftyone/blob/develop/app/packages/state/src/recoil/atoms.ts#L9

Reply:
👍